docs/UI/OnKeyboardInput/assets/KeyboardEventInputKeyMask.txt

```diff
@@ -0,0 +1,61 @@
+participant App
+participant SDL
+participant HMI
+
+note over App,HMI:<align:center> **//Pre-conditions://**\n**1.** appID application is successfully registered and active on HMI\n**2.** HMI supports masking input characters
+participant App
+participant SDL
+participant HMI
+
+note over App,HMI:<align:center> **//Pre-conditions://**\n**1.** appID application is successfully registered and active on HMI\n**2.** HMI supports masking input characters
+
+HMI->>SDL: BC.OnSystemCapabilityUpdated(**maskInputCharactersSupported=true**)
+SDL->>App: OnSystemCapabilityUpdated(maskInputCharactersSupported=true)
+opt App sends the request to mask keyboard input characters 
```

Comment:
This part of the diagram does not appear in the image. Please rerender it

Reply:
please, see https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/238/commits/79567634568f1cd0f457817866a0561a5a1a89aa